jobserv_worker.py

```diff
@@ -678,6 +716,7 @@ def get_args(args=None):
         help="""Worker name to register. If none is provided, the value of
                 /etc/hostname will be used.""",
     )
+    p.add_argument("--jwt", help="A JWT to use for API authentication")
```

Comment:
What entity does create the `jwt` token that is supposed to specified via `--jwt` token? I suppose, this is either a human or some automation script. if so, maybe it makes sense to extend `create_jwt` command so it actually creates a JWT token not just a key/cert pair.?

Reply:
I mean something that is done in the test below, e.g.
```
key, cert = create_jwt(["org1"])
        jobserv.worker_jwt._keys.clear()

        headers = {"kid": _keyid(cert)}
        worker = {"name": "MrJWT"}
        worker["exp"] = datetime.datetime.utcnow() + datetime.timedelta(seconds=30)
        encoded = jwt.encode(worker, key, algorithm="ES256", headers=headers)
```

Reply:
token creation needs to happen inside people's custom scaling logic. this just makes it a little easier to create a keypair that logic will depend on.